integration/test/basic/basic_test.go

```diff
@@ -31,7 +31,7 @@ func TestHelm(t *testing.T) {
 }
 
 func installCertManager(ctx context.Context, helmClient helmclient.Interface, logger micrologger.Logger) error {
-	tarballURL, err := appcatalog.GetLatestChart(ctx, catalogURL, certManagerAppName, "0.9.0")
+	tarballURL, err := appcatalog.GetLatestChart(ctx, catalogURL, certManagerAppName, "0.15.2")
```

Comment:
we could revert to this https://github.com/giantswarm/kiam-app/commit/35451946aeba1c691bb052d985c9556c467c21d2

as the specific version was added  to work with older kiam, having `""` will mean always get latest

Reply:
ok, that works for me :+1: